### ISSUE TITLE
Loss error during backward when using torch.nn.CrossEntropyLoss() with gpu

### ISSUE THREAD

**Issue by ambroiseodt** [ivadomed/MEEG-Brainstorm#18]
I am currently trying to use a gpu (cuda:0) on rosenberg server to run my code;
During training, the data are moved to the device cuda:0 but I have an error when doing the backward in the loss here https://github.com/AmbroiseOdonnat/MEEG-Brainstorm/blob/ao/seizure_classification/Train.py
I use torch.nn.CrossEntropyLoss() and I have the following error.
I precise that the code works perfectly on rosenberg and on my computer when I use a cpu. 
![Capture d’écran 2022-03-08 à 09 18 12](https://user-images.githubusercontent.com/64415312/157262325-0c5996d8-cd25-4944-9898-56bab0b8de39.png)
)

**Comment by ambroiseodt** [comment]
Similar to this issue https://github.com/pytorch/pytorch/issues/56747 
It seems to be an issue with CUDA 11.1 and linear layers / matrices multiplications.
I downgraded CUDA 11.1 to CUDA 10.2 and it works fine now.
